manifests/0000_70_dns-operator_00-cluster-role.yaml

```diff
@@ -134,4 +136,6 @@ rules:
   resources:
   - dnsnameresolvers/status
   verbs:
+  - get
   - update
+  - patch
```

Comment:
Adding these permissions for getting the events related to the dnsnameresolver status ([slack thread](https://redhat-internal.slack.com/archives/C04VCUBSU4U/p1714058335575259?thread_ts=1713964224.175029&cid=C04VCUBSU4U)).